### ISSUE TITLE
Public/Private IP

### ISSUE THREAD

**Issue by davidhood72** [MicrosoftDocs/OfficeDocs-Exchange#2537]
I think it might be useful if you could clarify if the IP addresses in the rule should be the public IP address of a client or its actual IP address.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b9cdf39f-da71-43ab-0ceb-a816bdc19d04
* Version Independent ID: 1fff6b28-f8d8-d9b3-1730-db2cac7835a0
* Content: [Client Access Rules in Exchange Online](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/client-access-rules/client-access-rules)
* Content Source: [Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/client-access-rules/client-access-rules.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/client-access-rules/client-access-rules.md)
* Service: **exchange-online**
* Technology: ****
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by scanum** [comment]
@davidhood72 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by get-itips** [comment]
Hello @davidhood72 How are you?
I tested this on my lab with Exchange Online (don't know if you are using Exchange On-Premises or Online) 
Blocked my Public IP and allowed my LAN IP using Client Access Rules.
Tried to open OWA and my connection was blocked, The more details on the error page informed me that the client's IP address was identified as the Public one. Tested this also on an Exchange Server, specified a LAN range, and blocked EAC, couldn't enter EAC from that IP range.

I hope this information helps you.
